scripts/host-only/wkdev-create

```diff
@@ -262,6 +262,10 @@ try_process_journal() {
 try_process_keyring() {
 
     local keyring_directory="${XDG_RUNTIME_DIR}/keyring"
+
+    # Respect $SSH_AUTH_SOCK (but strip the "unix:path=" prefix)
+    [ -v SSH_AUTH_SOCK ] && keyring_directory=$(dirname "${SSH_AUTH_SOCK##unix:path=}")
```

Comment:
Logic looks good to me. I'd just request not doing one-liners and instead have an explicit `if` statement.

Reply:
Done!